rpcserver.go

```diff
@@ -746,11 +746,22 @@ func (s *rpcServer) DepositAccount(ctx context.Context,
 			"minimum is %d sat/kw", feeRate, chainfee.FeePerKwFloor)
 	}
 
+	bestHeight := atomic.LoadUint32(&s.bestHeight)
+
+	// If provided, determine new expiration value.
+	var expiryHeight uint32
+	switch {
+	case req.GetAbsoluteExpiry() != 0:
```

Comment:
We also need to add the check for both being set, in case someone uses the API directly. Same for the withdrawal.